src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -41,13 +43,17 @@ def load_arguments_eh(self, _):
         c.argument('capacity', type=int, help='Capacity for Sku')
         c.argument('is_auto_inflate_enabled', options_list=['--enable-auto-inflate'], arg_type=get_three_state_flag(), help='A boolean value that indicates whether AutoInflate is enabled for eventhub namespace.')
         c.argument('maximum_throughput_units', type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
+        c.argument('default_action', arg_group='networkrule', options_list=['--default-action'], arg_type=get_enum_type(['Allow', 'Deny']),
+                   help='Default Action for Network Rule Set. Possible values include: Allow, Deny')
 
     with self.argument_context('eventhubs namespace update') as c:
         c.argument('tags', arg_type=tags_type)
         c.argument('sku', options_list=['--sku'], arg_type=get_enum_type(SkuName))
         c.argument('capacity', type=int, help='Capacity for Sku')
         c.argument('is_auto_inflate_enabled', options_list=['--enable-auto-inflate'], arg_type=get_three_state_flag(), help='A boolean value that indicates whether AutoInflate is enabled for eventhub namespace.')
         c.argument('maximum_throughput_units', type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
+        c.argument('default_action', arg_group='networkrule', options_list=['--default-action'], arg_type=get_enum_type(['Allow', 'Deny']),
+                   help='Default Action for Network Rule Set. Possible values include: Allow, Deny')
```

Comment:
Same here. Remove the "Possible values..." text to avoid duplication.

Reply:
resolved